crates/swc_ecma_transforms_compat/src/es2020/nullish_coalescing/mod.rs

```diff
@@ -47,12 +58,16 @@ impl VisitMut for NullishCoalescing {
 
     /// Prevents #1123
     fn visit_mut_block_stmt(&mut self, s: &mut BlockStmt) {
```

Comment:
Is the purpose of this change is passing `Config`?
I think creating a new visitor is better. (Generally, because of readability and the possibility of adding fields.)

Reply:
What do you mean by creating a new visitor? also you seems to be referencing a non changed line